modules/api/src/main/java/org/apache/ignite/sql/ColumnType.java

```diff
@@ -90,7 +90,10 @@ public enum ColumnType {
     NUMBER,
 
     /** Null. */
-    NULL;
+    NULL,
+
+    /** Binary. */
+    BINARY;
```

Comment:
I don't think we should add a new SQL column type for this. Let's use existing `BYTE_ARRAY`.